### PR TITLE
[9.0][ADD] stock_inventory_chatter

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Request of this module: https://github.com/OCA/stock-logistics-warehouse/issues/288

Stock Inventory Chatter
=======================

This module adds the capability to log the changes being done in Inventory Adjustments.

### CODE REVIEW

**Review by andhit-r — APPROVED** [review]

**Review by elicoidal — APPROVED** [review]
LGTM

File: stock_inventory_chatter/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from openerp import api, fields, models
+
+
+class StockInventory(models.Model):
+    _name = 'stock.inventory'
+    _inherit = ['stock.inventory', 'mail.thread', 'ir.needaction_mixin']
+
+    partner_id = fields.Many2one(track_visibility='always')
+    state = fields.Selection(track_visibility='onchange')
```
Comment (LoisRForgeFlow) [inline stock_inventory_chatter/models/stock.py:14]: I would also track `location_id` and `filter`

File: stock_inventory_chatter/models/stock.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    @api.multi
+    def _track_subtype(self, init_values):
+        self.ensure_one()
+        if 'state' in init_values and self.state == 'confirm':
+            return 'stock_inventory_chatter.mt_inventory_confirmed'
+        elif 'state' in init_values and self.state == 'done':
+            return 'stock_inventory_chatter.mt_inventory_done'
```
Comment (LoisRForgeFlow) [inline stock_inventory_chatter/models/stock.py]: This msgs seems that are not being logged.

**Review by LoisRForgeFlow — COMMENTED** [review]
Some little comments

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
messages mt_inventory_confirmed and mt_inventory_done not being logged in runbot

File: stock_inventory_chatter/models/stock.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply stock_inventory_chatter/models/stock.py]: I think this msgs are sent through mail to the followers of the thread.

File: stock_inventory_chatter/models/stock.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class StockInventory(models.Model):
+    _name = 'stock.inventory'
+    _inherit = ['stock.inventory', 'mail.thread', 'ir.needaction_mixin']
```
Comment (AaronHForgeFlow) [inline stock_inventory_chatter/models/stock.py:11]: is it really needed to inherit form ir.needaction_mixin?

File: stock_inventory_chatter/models/stock.py (hunk 2 above)
Comment (AaronHForgeFlow) [reply stock_inventory_chatter/models/stock.py]: In that case I'd also track the cancelled state @mreficent

File: stock_inventory_chatter/models/stock.py (hunk 3 above)
Comment (MiquelRForgeFlow) [reply stock_inventory_chatter/models/stock.py:11]: It seems so.

**Comment by MiquelRForgeFlow** [comment]
Comments attended.

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Thanks

**Pull request merged**